recipes/gaia-ecs/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package LANGUAGES CXX)
+
+find_package(gaia REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE gaia::gaia)
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)
+set(THREADS_PREFER_PTHREAD_FLAG ON)
+find_package(Threads REQUIRED)
+target_link_libraries(${PROJECT_NAME} PRIVATE Threads::Threads)
```

Comment:
The fact that he test package isn't using threads directly indicates that something is missing in the `package_info` method. Perhaps:

```
if self.settings.os in ["FreeBSD", "Linux"]:
    self.cpp_info.system_libs = ["pthread"]
```

Reply:
Good point!